recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
I've been persuaded. Moving this to conda-forge-build-setup seems like the best way forward.